numaprom/udf/window.py

```diff
@@ -47,9 +44,15 @@ def __aggregate_window(
     Returns an empty list if adding the element does not create a new entry
     to the set.
     """
+    redis_conf = ConfigManager().get_redis_config()
```

Comment:
These are all class methods right?
We can just do maybe,
```suggestion
    redis_conf = ConfigManager.get_redis_config()
```
and avoid object instantiation